load-test-framework/src/main/resources/gce/cps-gcloud-go-publisher_startup_script.sh

```diff
@@ -0,0 +1,38 @@
+#/bin/bash
+
+#######################################
+# Query GCE for a provided metadata field.
+# See https://developers.google.com/compute/docs/metadata
+# Globals:
+#   None
+# Arguments:
+#   $1: The path to the metadata field to retrieve
+# Returns:
+#   The value stored at the metadata field
+#######################################
+function metadata() {
+  curl --silent --show-error --header 'Metadata-Flavor: Google' \
+    "http://metadata/computeMetadata/v1/${1}";
+}
+
+readonly TMP="$(mktemp -d)"
+readonly BUCKET=$(metadata instance/attributes/bucket)
+
+[[ "${TMP}" != "" ]] || error mktemp failed
```

Comment:
This was copied from a different loadtest script we had written. It is not a command, it just causes the script to exit since the error command doesn't exist. We can change this to something else, but from the load test it will just look like a byzantine error regardless.